pydra/engine/tests/test_specs.py

```diff
@@ -101,6 +101,10 @@ def __init__(self):
         class Result:
             def __init__(self):
                 self.output = Output()
+                self.errored = False
+
+            def get_output_field(self, field):
+                return getattr(self.output, field)
```

Comment:
since this is one line replacing another line, is there a need for this function?

Reply:
these are "fake" classes for testing, so need the same methods. I'm not completely sure if I like these classes/tests, but left for now